test/integration/main_test.go

```diff
@@ -0,0 +1,23 @@
+package integration
+
+import (
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/yahoo/k8s-athenz-istio-auth/test/integration/framework"
+)
+
+func TestMain(m *testing.M) {
+	f, err := framework.Setup()
+	if err != nil {
+		log.Println("Error setting up test framework:", err)
+		os.Exit(1)
+	}
+
+	time.Sleep(time.Second * 15)
+	exitCode := m.Run()
+	f.Teardown()
```

Comment:
can we call `CreateAthenzDomain()` after m.Run()

Reply:
Yep, I added it but we will probably remove with the next PR.